### PR TITLE
SchemaDownlader: Update to download deprecated input fields

### PR DESCRIPTION
**Pull request by ndwhelan**
Added `(includeDeprecated: true)` as a params to `inputFields` in the `fragment` portion of the GraphQL query to download a schema.

### Current Behavior

When the `downloadApolloSchema` Gradle task is run, input fields with the `@deprecated` tag **ARE NOT** downloaded.

### Expected Change in Behavior

When the `downloadApolloSchema` Gradle task is run, input fields with the `@deprecated` tag **ARE** downloaded.

### CODE REVIEW

File: libraries/apollo-tooling/src/main/kotlin/com/apollographql/apollo3/tooling/SchemaDownloader.kt [hunk 1]
```diff
@@ -199,7 +199,7 @@ object SchemaDownloader {
         isDeprecated
         deprecationReason
       }
-      inputFields {
+      inputFields(includeDeprecated: true {
```
Comment (martinbonnin) [inline libraries/apollo-tooling/src/main/kotlin/com/apollographql/apollo3/tooling/SchemaDownloader.kt:202]: Looks like you're missing a closing parenthesis here

File: libraries/apollo-tooling/src/main/kotlin/com/apollographql/apollo3/tooling/SchemaDownloader.kt [hunk 2]
```diff
@@ -199,7 +199,7 @@ object SchemaDownloader {
...
         ...InputValue
```
Comment (martinbonnin) [inline libraries/apollo-tooling/src/main/kotlin/com/apollographql/apollo3/tooling/SchemaDownloader.kt:203]: You'll also want to update the `InputValue` fragment to include `isDeprecated` and `deprecationReason`

**Comment by ndwhelan** [comment]
Wow, that is embarrassing. Sorry for that. I need to actually pull the code
and do this properly instead of just using the GitHuh editor. Thank you for
the advice.

**Comment by martinbonnin** [comment]
Hi @ndwhelan how can we help here? Want us to get this through the finish line?

**Comment by ndwhelan** [comment]
Apologies. Numerous high priority things came up for me this week. I'd like
to try to finish it this weekend.

If you think it's preferable that you take it on at this point, though, I
certainly would not have any problem with that.

Best,

Nick

**Comment by martinbonnin** [comment]
No worries at all! Take your time, just wanted to make sure this doesn't fall through the cracks.

**Comment by ndwhelan** [comment]
I've updated things from the PR, and updated a test. The tests are passing, but I wasn't really convinced it wouldn't return the whole thing regardless of my query in SchemaDownloader. I want to say it was, because it would fail if I put bad syntax in, but I'm not 100% sure. Any feedback on the test or improving that would be appreciated. I should be able to resolve it promptly if necessary.

File: libraries/apollo-tooling/src/main/kotlin/com/apollographql/apollo3/tooling/SchemaDownloader.kt (hunk 2 above)
Comment (ndwhelan) [reply libraries/apollo-tooling/src/main/kotlin/com/apollographql/apollo3/tooling/SchemaDownloader.kt:203]: Done.

**Review by martinbonnin — APPROVED** [review]
This all looks very good to me, Thanks! 🙏

**Review by BoD — APPROVED** [review]
👍

**Comment by BoD** [comment]
This has been released as part of [v3.7.5](https://github.com/apollographql/apollo-kotlin/releases/tag/v3.7.5)

**Pull request merged**